source/containers.tex

```diff
@@ -18364,7 +18364,7 @@
 
 \rSec4[span.iterators]{Iterator support}
 
-\indexlibrarymember{iterator}{span}%
+\indexlibrarymember{span}{iterator}%
```

Comment:
This is not what we usually do (globally); please split off into a separate commit.

Reply:
Reverted this drive-by, no separate commit forthcoming at this time.